templates/books/book-detail.html

```diff
@@ -98,6 +100,9 @@ <h1 class="h2">{{ book.title }}</h1>
                             Агучка: удакладняецца...
                         {% endif %}
                         <br>
+                        {% if narration.duration %}
```

Comment:
We should render this only when there are two or more narrations. So need to add an if similar to above.

Reply:
Ah, nvm. Didn't notice that this is already in that "> narrations" section.